lib/rubycritic/analysers/helpers/reek.rb

```diff
@@ -1,15 +1,6 @@
 require "reek"
-require "reek/configuration/app_configuration"
-require "ostruct"
 
 module Rubycritic
-
   class Reek < ::Reek::Examiner
```

Comment:
Yes, but now that you mention it, this reeks (pun intended) of premature optimization. I removed the whole class.